build.sbt

```diff
@@ -35,7 +35,9 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .crossType(CrossType.Pure)
   .settings(name := "cats-mtl")
   .settings(
-    libraryDependencies += "org.typelevel" %%% "cats-core" % CatsVersion
+    libraryDependencies += "org.typelevel" %%% "cats-core" % CatsVersion,
+    libraryDependencies += "org.typelevel" %%% "cats-effect" % "3.4.5",
+
```

Comment:
Adding CE would make a circular dependency. Also, why? 🤔

Reply:
Because this was my initial playground experimenting with IOLocal and I didn't clean up after myself.

[hangs head in shame]